plugins/discovery/discovery_test.go

```diff
@@ -114,7 +114,7 @@ func TestProcessBundle(t *testing.T) {
 			"config": {
 				"bundle": {"name": "test1"},
 				"status": {},
-				"decision_logs": {}
```

Comment:
Do these test cases still need to change? I figured this was due to the original config change which we backed out. I just want to make sure that discovery bundles do not have to redefine services or include pointers to implicitly defined services.

Reply:
Ah yea, let me unwind these. Might add in a couple more explicit tests for the new checks too.